tests/foreman/cli/test_template.py

```diff
@@ -249,111 +249,58 @@ def test_positive_clone(self):
         new_template = Template.info({'id': result[0]['id']})
         self.assertEqual(new_template['name'], cloned_template_name)
 
-    # Take Templates out of Tech Preview Feature Tests
 
-    @stubbed
-    @tier3
-    def test_positive_import_log_to_production(self):
-        """Assert template import logs are logged to production logs
 
-        :id: aeb0b80a-1eb9-4dfe-a67a-7c76ee51f61d
+class TestTemplateSyncTestCase:
 
-        :Steps:
-            1. Using hammer, Import template from a source
-
-        :expectedresults:
-            1. Assert template import task and status logged to production log
-
-        :Requirement: Take Templates out of tech preview
-
-        :CaseAutomation: NotAutomated
-
-        :CaseLevel: System
-
-        :CaseAutomation: NotAutomated
-        """
-
-    @stubbed
-    @tier3
-    def test_positive_export_log_to_production(self):
-        """Assert template export logs are logged to production logs
-
-        :id: 55dca9ce-18b1-4846-bae9-44a3b94a2749
-
-        :Steps:
-            1. Using hammer, Export template to destination
-
-        :expectedresults:
-            1. Assert template export task and status logged to production log
-
-        :Requirement: Take Templates out of tech preview
-
-        :CaseAutomation: NotAutomated
-
-        :CaseLevel: System
-
-        :CaseAutomation: NotAutomated
-        """
-
-    @stubbed
-    @tier3
-    def test_rake_templates_import_depreciated(self):
-        """Assert running `foreman-rake templates:import` shows depreciation
-        message
+    @tier2
+    def test_negative_import_locked_template(self, module_org, create_import_export_local_dir):
+        """Assure locked templates are not pulled from repository.
 
-        :id: 75aab2b2-fcbb-436c-924d-4897d49ee0d2
+        :id: 88e21cad-448e-45e0-add2-94493a1319c5
 
         :Steps:
-            1. Run `foreman-rake templates:import` task through
-            command from a directory having no templates to import
+            1. Using nailgun or direct API call try to import a locked template
```

Comment:
Not sure do we required to mention that we are using nailgun?

Reply:
Its Really not necessary, but the tests were written from Bruno and I don't want to touch them.

Maybe a different PR that removes these docstrings can be quicker. But not something that `have to do`.